MinecraftClient/McClient.cs

```diff
@@ -215,7 +216,7 @@ private void StartClient(string user, string uuid, string sessionID, string serv
                     if (Settings.ReplayMod_Enabled) { BotLoad(new ReplayCapture(Settings.ReplayMod_BackupInterval)); }
 
                     //Add your ChatBot here by uncommenting and adapting
-                    //BotLoad(new ChatBots.YourBot());
+                    // BotLoad(new ChatBots.YourBot());

```

Comment:
Just to avoid unnecessary change in PR:
```suggestion
                    //BotLoad(new ChatBots.YourBot());

```